NovaPC_Melhorenvio/app/code/community/NovaPC/Melhorenvio/Helper/Data.php

```diff
@@ -109,6 +109,12 @@ function formatCnpjCpf($value)
 		return preg_replace("/(\d{2})(\d{3})(\d{3})(\d{4})(\d{2})/", "\$1.\$2.\$3/\$4-\$5", $cnpj_cpf);
 	}
 
+	public function isCNPJ($valor){

```

Comment:
Ajustar identação do método.

Reply:
Ajuste realizado no commit "Correction of indentation"